docs/resources/schema-manager.md

```diff
@@ -37,30 +29,57 @@ Example command line usage:
+- This table holds all schema versions that have been applied to the database.
+
+**InstanceSchema**
+- Each DICOM instance reports the schema version it is at, as well as the versions it is compatible with, to the InstanceSchema database table.
 
 ------------
 
-#### Important Database Tables
-SchemaVersion
-- This table holds all schema versions that have ben applied to the database.
+### Terminology
 
-InstanceShema
-- Each DICOM instance reports the schema version it is at, as well as the versions it is compatible with, to the InstanceSchema database table.
+**Current database version**
+- The maximum SchemaVersion version in the database. In the event this table is empty, the version is 0.
+
+**Current instance version**
+- The maximum SchemaVersion version in the database that falls at or below the SchemaVersionConstants.Max value. For example, if the current database version is 25, but SchemaVersionConstants.Max is 23, the instance's current version will be 23.
+
+**Available version**
+- Any version greater than or equal to the current database version.
+
+**Compatible version**
+- Any version from SchemaVersionConstants.Min to SchemaVersionConstants.Max (inclusive).
 
 ------------
 
-#### How does it work?
+### How does it work?
 
 Schema Manager runs through the following steps:
 1. Verifies all arguments are supplied and valid.
-2. Calls the [healthcare-shared-components ApplySchema function](https://github.com/microsoft/healthcare-shared-components/blob/20506ffba19905abe882812a25d74866d1e1dcb0/src/Microsoft.Health.SqlServer/Features/Schema/Manager/SqlSchemaManager.cs#L53), which:
+2. Calls the [healthcare-shared-components ApplySchema function](https://github.com/microsoft/healthcare-shared-components/blob/main/src/Microsoft.Health.SqlServer/Features/Schema/Manager/SqlSchemaManager.cs#L53), which:
 	1. Ensures the base schema exists.
 	2. Ensures instance schema records exist.
 		1. Since DICOM Server implements its own ISchemaClient (DicomSchemaClient), if there are no instance schema records, the upgrade continues uninterrupted. In healthcare-shared-components, this would throw an exception and cancel the upgrade.
 	3. Gets all available versions and compares them against all compatible versions.
-	4. Branches based on the current version:
-		1. If the current version is 1, only applies the latest full migration script.
-		2. If the current version is > 1, applies each available version (excluding the current version) one at a time until the database's schema version reaches the desired version input by the user (latest, next, or a specific version).
+	4. Based on the current schema version:
+		1. If there is no version (base schema only), the latest full migration script is applied.
+		2. If the current version is >= 1, each available version is applied (excluding the current version) one at a time until the database's schema version reaches the desired version input by the user (latest, next, or a specific version).
```

Comment:
you shouldn't need the `(excluding the current version)` clause if you change the available version definition